<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="4" ignored="0" total="5" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="My Sample Suite" duration-ms="29397" started-at="2018-09-23T15:30:07Z" finished-at="2018-09-23T15:30:37Z">
    <groups>
    </groups>
    <test name="Delete Portfolio test" duration-ms="12212" started-at="2018-09-23T15:30:07Z" finished-at="2018-09-23T15:30:20Z">
      <class name="com.qtpselenium.hybrid.testcases.DeletePortfolioTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.hybrid.testcases.DeletePortfolioTest@5ccddd20]" name="init" is-config="true" duration-ms="259" started-at="2018-09-23T21:00:07Z" finished-at="2018-09-23T21:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="deletePortfolioTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.testcases.DeletePortfolioTest@5ccddd20]" name="deletePortfolioTest" duration-ms="11685" started-at="2018-09-23T21:00:08Z" data-provider="getData" finished-at="2018-09-23T21:00:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Runmode=Y, Portfolioname=SamplePortfolio12, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element not found password_id]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element not found password_id
	at org.testng.Assert.fail(Assert.java:94)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:166)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:153)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:82)
	at com.qtpselenium.hybrid.keywords.ApplicationKeywords.login(ApplicationKeywords.java:49)
	at com.qtpselenium.hybrid.keywords.ApplicationKeywords.defaultLogin(ApplicationKeywords.java:40)
	at com.qtpselenium.hybrid.driver.DriverScript.executeTest(DriverScript.java:64)
	at com.qtpselenium.hybrid.testcases.DeletePortfolioTest.deletePortfolioTest(DeletePortfolioTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletePortfolioTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.testcases.DeletePortfolioTest@5ccddd20]" name="quit" is-config="true" duration-ms="218" started-at="2018-09-23T21:00:19Z" finished-at="2018-09-23T21:00:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qtpselenium.hybrid.testcases.DeletePortfolioTest -->
    </test> <!-- Delete Portfolio test -->
    <test name="Create PortFolio test" duration-ms="12351" started-at="2018-09-23T15:30:07Z" finished-at="2018-09-23T15:30:20Z">
      <class name="com.qtpselenium.hybrid.testcases.CreatePortfolioTest">
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.hybrid.testcases.CreatePortfolioTest@25d250c6]" name="init" is-config="true" duration-ms="259" started-at="2018-09-23T21:00:07Z" finished-at="2018-09-23T21:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="createPortFolioTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.testcases.CreatePortfolioTest@25d250c6]" name="createPortFolioTest" duration-ms="10676" started-at="2018-09-23T21:00:08Z" data-provider="getData" finished-at="2018-09-23T21:00:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{PortFolioName=SamplePortfolio12, Runmode=Y, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element not found password_id]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element not found password_id
	at org.testng.Assert.fail(Assert.java:94)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:166)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:153)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:82)
	at com.qtpselenium.hybrid.keywords.ApplicationKeywords.login(ApplicationKeywords.java:49)
	at com.qtpselenium.hybrid.keywords.ApplicationKeywords.defaultLogin(ApplicationKeywords.java:40)
	at com.qtpselenium.hybrid.driver.DriverScript.executeTest(DriverScript.java:64)
	at com.qtpselenium.hybrid.testcases.CreatePortfolioTest.createPortFolioTest(CreatePortfolioTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPortFolioTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.testcases.CreatePortfolioTest@25d250c6]" name="quit" is-config="true" duration-ms="1011" started-at="2018-09-23T21:00:18Z" finished-at="2018-09-23T21:00:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.hybrid.testcases.CreatePortfolioTest@25d250c6]" name="init" is-config="true" duration-ms="215" started-at="2018-09-23T21:00:19Z" finished-at="2018-09-23T21:00:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="SKIP" signature="createPortFolioTest(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.testcases.CreatePortfolioTest@25d250c6]" name="createPortFolioTest" duration-ms="1" started-at="2018-09-23T21:00:20Z" data-provider="getData" finished-at="2018-09-23T21:00:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{PortFolioName=SamplePortfolio12, Runmode=N, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test as Runmode is NO]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test as Runmode is NO
	at com.qtpselenium.hybrid.testcases.CreatePortfolioTest.createPortFolioTest(CreatePortfolioTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPortFolioTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.testcases.CreatePortfolioTest@25d250c6]" name="quit" is-config="true" duration-ms="139" started-at="2018-09-23T21:00:20Z" finished-at="2018-09-23T21:00:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qtpselenium.hybrid.testcases.CreatePortfolioTest -->
    </test> <!-- Create PortFolio test -->
    <test name="Login Test" duration-ms="29397" started-at="2018-09-23T15:30:07Z" finished-at="2018-09-23T15:30:37Z">
      <class name="com.qtpselenium.hybrid.testcases.LoginTest">
        <test-method status="FAIL" signature="doLogin(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.testcases.LoginTest@7995092a]" name="doLogin" duration-ms="19840" started-at="2018-09-23T21:00:08Z" data-provider="getData" finished-at="2018-09-23T21:00:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExectedResult=LoginSuccess, Username=ashishthakur1983, Password=pass@1234, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element not found password_id]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element not found password_id
	at org.testng.Assert.fail(Assert.java:94)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:166)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:153)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:82)
	at com.qtpselenium.hybrid.driver.DriverScript.executeTest(DriverScript.java:56)
	at com.qtpselenium.hybrid.testcases.LoginTest.doLogin(LoginTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.hybrid.testcases.LoginTest@7995092a]" name="init" is-config="true" duration-ms="0" started-at="2018-09-23T21:00:08Z" finished-at="2018-09-23T21:00:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.testcases.LoginTest@7995092a]" name="quit" is-config="true" duration-ms="111" started-at="2018-09-23T21:00:28Z" finished-at="2018-09-23T21:00:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method status="PASS" signature="init()[pri:0, instance:com.qtpselenium.hybrid.testcases.LoginTest@7995092a]" name="init" is-config="true" duration-ms="1" started-at="2018-09-23T21:00:28Z" finished-at="2018-09-23T21:00:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="doLogin(java.util.Hashtable)[pri:0, instance:com.qtpselenium.hybrid.testcases.LoginTest@7995092a]" name="doLogin" duration-ms="8828" started-at="2018-09-23T21:00:28Z" data-provider="getData" finished-at="2018-09-23T21:00:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{ExectedResult=LoginFailure, Username=ashishthakur1983, Password=XXXXX, Runmode=Y, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element not found password_id]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element not found password_id
	at org.testng.Assert.fail(Assert.java:94)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:166)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:153)
	at com.qtpselenium.hybrid.keywords.GenericKeywords.type(GenericKeywords.java:82)
	at com.qtpselenium.hybrid.driver.DriverScript.executeTest(DriverScript.java:56)
	at com.qtpselenium.hybrid.testcases.LoginTest.doLogin(LoginTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.qtpselenium.hybrid.testcases.LoginTest@7995092a]" name="quit" is-config="true" duration-ms="253" started-at="2018-09-23T21:00:37Z" finished-at="2018-09-23T21:00:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.qtpselenium.hybrid.testcases.LoginTest -->
    </test> <!-- Login Test -->
  </suite> <!-- My Sample Suite -->
</testng-results>
